pkg/controller/argocd/argocd.go

```diff
@@ -14,6 +14,15 @@ type resource struct {
 	Clusters  []string `json:"clusters"`
 }
 
+// OIDCConfig enables Integration with RHSSO
+type OIDCConfig struct {
+	Name           string   `json:"name"`
+	Issuer         string   `json:"issuer"`
+	ClientID       string   `json:"clientID"`
+	ClientSecet    string   `json:"clientSecret"`
```

Comment:
Please fix typo

Reply:
@wtam Can you point me to the typo ?

Reply:
ClientSecet

Reply:
Sure .. Thanks @wtam2018  .. Also I see a minor issue with unit tests .. will fix it once I am back .

Reply:
My other comment on this is that do any of these fields allow empty (omitempty)?